content/guides/core-concepts/cypress-app.md

```diff
@@ -6,29 +6,108 @@ title: Cypress App
 - How to use the Selector Playground for targeting your page elements
-- How to debug tests using the built-in features of the Cypress App
+- How to debug tests using the built-in features of the Cypress app
 
 </Alert>
 
 ## Overview
 
-Cypress runs tests in a unique interactive application that allows you to see
-commands as they execute while also viewing the Application or Component Under
-Test.
+The Cypress desktop app has powerful features for creating, configuring,
+browsing, and most importantly running your tests. We'll touch on a few of the
+different functions before moving on to the Test Runner, the heart of the
+Cypress testing experience.
 
-<DocsImage src="/img/guides/cypress-app/gui-diagram.png" alt="Cypress App"></DocsImage>
+<Alert type="info">
+
+<strong class="alert-header">Dashboard Integration</strong>
+
+A number of the features described below will only be shown if you are logged
+into the [Cypress Dashboard](/guides/dashboard/introduction). If you're not sure
+whether you're logged in, you can always check using the profile control in the
+top right of the window.
+
+As the data used by these features are pulled from our cloud servers, they will
+only reflect test results that were
+[recorded](/guides/dashboard/projects#Set-up-a-project-to-record).
+
+</Alert>
+
+### The Launchpad
+
+<DocsImage src="/img/guides/core-concepts/cypress-app/the-launchpad.png" alt="The Launchpad"></DocsImage>
+
+The Launchpad is your portal to Cypress, helping with onboarding, choosing a
+testing type and launching a browser.
+
+Once you get comfortable with Cypress you might find you don't need the
+Launchpad any more. In this case
+[you can run `cypress open` with the `--browser` and `--component` or `--e2e` options](/guides/guides/command-line#cypress-open)
+to go straight to the Spec Explorer.
+
+### The Spec Explorer
+
+TODO Image with rest of columns
+<DocsImage src="/img/guides/core-concepts/cypress-app/spec-explorer.png" alt="The Spec Explorer"></DocsImage>
+
+On choosing your browser in the Launchpad, you'll be presented with a list of
+your specs, their names, locations, and associated metadata columns. Here you
+can launch specs by clicking them, create new blank or example specs, or search
+specs by name (handy for large test suites).
+
+There are currently three spec metadata columns.
+
+#### Last updated
+
+This column shows when the code of this spec was last updated, using your local
+system's own file metadata. This metadata will vary from machine to machine, as
+it reflects the state of the code as stored on that computer.
+
+#### Latest runs
+
+This column shows the last time the spec was run, and what status was returned.
+This metadata comes from the Dashboard (see [note above](#Overview)) so should
+be the same everywhere.
+
+#### Average duration
+
+This column shows the average time taken to run the entire spec file. Again,
+this metadata comes from the Dashboard.
```

Comment:
It may be helpful to explain that the average is over the last 4 runs for the current branch. Also, in case the spec had multiple groups, the average will reflect the average across runs for the average of durations for groups in each run.